source/github-events/on-new-comments.ts

```diff
@@ -57,7 +68,7 @@ function addListeners(): void {
 	delegates.add(delegate(document, '.js-ajax-pagination', 'submit', paginationSubmitHandler));
 
 	// Collapsed comments are loaded later using an include-fragment element
-	delegates.add(delegate(document, 'details.js-comment-container include-fragment', 'loadstart', getFragmentLoadHandler(run), true));
+	delegates.add(delegate(document, 'details.js-comment-container include-fragment:not([class])', 'loadstart', getFragmentLoadHandler(run), true));
```

Comment:
Collapsed review comments aren't loaded with their dropdown anymore.

Instead, they're created with two `include-fragment` that will be loaded when hovering on the dropdown button (the three dots). This selector excludes those.